test/notificationSpec.js

```diff
@@ -1013,6 +1014,13 @@ describe('Notifications', function () {
       })
     })
   })
+
+  describe('parseLedger', function () {
```

Comment:
What about adding a testing with a ledger that has a port number, instead of unit testing a private function?